plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/ray_launcher.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import logging
+import os
+import tempfile
+from pathlib import Path
+from subprocess import PIPE, Popen
+from typing import Any, Dict, Sequence
+
+import cloudpickle
```

Comment:
Cloudpickle is also part of ray as `ray.cloudpickle`: https://github.com/ray-project/ray/tree/master/python/ray/cloudpickle

Not sure what modifications they made (if any)

Reply:
we can ask them. if I had to guess this is just to be sure that the control the cloudpickle version they are using internally.